course_access_groups/monkeypatch.py

```diff
@@ -0,0 +1,37 @@
+"""
+Monkey patching to address missing features from the Open edX plugin system.
+
+Until Open edX gets pluggable course fields (imho it should), we'll have this monkey patching.
+"""
+
+from __future__ import absolute_import, unicode_literals
+import logging
+from xblock.fields import List, Scope
+from xmodule.course_module import CourseFields
+from django.utils.translation import ugettext as _
+
+log = logging.getLogger(__name__)
+
+
+def add_to_course_fields():
+    """
+    Add the `course_access_groups` field to the CourseFields class.
+
+    This makes every course has the new `course_access_groups` field without the need to hard code such logic for
```

Comment:
What is in the `course_access_groups` field? Looks like a list. What is in the list?

Reply:
@johnbaldwin it's the field below, please see lines 29-37 it has documentation embedded:

```python
CourseFields.course_access_groups = List(
    display_name=_('Course Access Groups'),
    help=_(
        'The IDs for the Course Access Groups which this course is available to. Enter the IDs in the '
        'following JSON array format: ["employees", "customers", "university"]'
    ),
    default=[],
    scope=Scope.settings
)
```